src/worker.jl

```diff
@@ -200,21 +221,70 @@ function worker_init(f::File)
             return take!(buf)
         end
 
-        function render_mimetypes(value)
+        # TODO: where is this key?
+        function _to_format()
+            fm = FRONTMATTER[]
+            if haskey(fm, "pandoc")
+                pandoc = fm["pandoc"]
+                if isa(pandoc, Dict) && haskey(pandoc, "to")
+                    to = pandoc["to"]
+                    isa(to, String) && return to
+                end
+            end
+            return nothing
+        end
```

Comment:
@jkrumbiegel where was the `to` format expected to be in this data?

Reply:
Here's a full options json for reference. The setting is grouped under `format`:

```js
{
  julia_cmd: "julia",
  oneShot: false,
  supervisor_pid: undefined,
  target: {
    source: "/Users/krumbiegel/temp/quarto-tests/projecttest/cairomakie.qmd",
    input: "/Users/krumbiegel/temp/quarto-tests/projecttest/cairomakie.qmd",
    markdown: {
      value: "---\n" +
        "title: Quarto test\n" +
        "format:\n" +
        "  html: default\n" +
        "  pdf:\n" +
        "    include-in-header:\n" +
        "      - text: |\n" +
        "       "... 1401 more characters,
      map: [Function: map]
    },
    metadata: {
      title: "Quarto test",
      format: {
        html: "default",
        pdf: { "include-in-header": [Array] },
        typst: "default"
      },
      julia: { exeflags: [ "--project=." ] },
      "fig-width": 5
    }
  },
  resourceDir: "/Users/krumbiegel/dev/quarto-cli/src/resources/",
  tempDir: "/var/folders/z5/r5q6djwn5g10k3w279bn37700000gn/T/quarto-session69317aa6/19de90d7/2446b186",
  dependencies: true,
  libDir: "cairomakie_files/libs",
  format: {
    identifier: {
      "display-name": "HTML",
      "target-format": "html",
      "base-format": "html"
    },
    execute: {
      "fig-width": 5,
      "fig-height": 5,
      "fig-format": "retina",
      "fig-dpi": 96,
      "fig-asp": undefined,
      "mermaid-format": undefined,
      "df-print": "default",
      error: false,
      eval: true,
      cache: null,
      freeze: false,
      echo: true,
      output: true,
      warning: true,
      include: true,
      "keep-md": false,
      "keep-ipynb": false,
      ipynb: null,
      enabled: null,
      daemon: null,
      "daemon-restart": false,
      debug: true,
      "ipynb-filters": [],
      "ipynb-shell-interactivity": null,
      "plotly-connected": true
    },
    render: {
      "keep-tex": false,
      "keep-typ": false,
      "keep-source": false,
      "keep-hidden": false,
      "prefer-html": false,
      "output-divs": true,
      "output-ext": "html",
      "fig-align": "default",
      "fig-pos": null,
      "fig-env": null,
      "code-fold": "none",
      "code-overflow": "scroll",
      "code-link": false,
      "code-line-numbers": false,
      "code-tools": false,
      "tbl-colwidths": "auto",
      "merge-includes": true,
      "inline-includes": false,
      "preserve-yaml": false,
      "latex-auto-mk": true,
      "latex-auto-install": true,
      "latex-clean": true,
      "latex-min-runs": 1,
      "latex-max-runs": 10,
      "latex-makeindex": "makeindex",
      "latex-makeindex-opts": [],
      "latex-tlmgr-opts": [],
      "latex-input-paths": [],
      "latex-output-dir": null,
      "link-external-icon": false,
      "link-external-newwindow": false,
      "self-contained-math": false,
      "format-resources": [],
      variant: undefined,
      "notebook-links": true
    },
    pandoc: {
      standalone: true,
      wrap: "none",
      "default-image-extension": "png",
      to: "html",
      toc: true,
      "number-sections": true,
      "output-file": "cairomakie.html"
    },
    language: {
      "toc-title-document": "Table of contents",
      "toc-title-website": "On this page",
      "related-formats-title": "Other Formats",
      "related-notebooks-title": "Notebooks",
      "source-notebooks-prefix": "Source",
      "other-links-title": "Other Links",
      "code-links-title": "Code Links",
      "launch-dev-container-title": "Launch Dev Container",
      "launch-binder-title": "Launch Binder",
      "article-notebook-label": "Article Notebook",
      "notebook-preview-download": "Download Notebook",
      "notebook-preview-download-src": "Download Source",
      "notebook-preview-back": "Back to Article",
      "manuscript-meca-bundle": "MECA Bundle",
      "section-title-abstract": "Abstract",
      "section-title-appendices": "Appendices",
      "section-title-footnotes": "Footnotes",
      "section-title-references": "References",
      "section-title-reuse": "Reuse",
      "section-title-copyright": "Copyright",
      "section-title-citation": "Citation",
      "appendix-attribution-cite-as": "For attribution, please cite this work as:",
      "appendix-attribution-bibtex": "BibTeX citation:",
      "title-block-author-single": "Author",
      "title-block-author-plural": "Authors",
      "title-block-affiliation-single": "Affiliation",
      "title-block-affiliation-plural": "Affiliations",
      "title-block-published": "Published",
      "title-block-modified": "Modified",
      "title-block-keywords": "Keywords",
      "callout-tip-title": "Tip",
      "callout-note-title": "Note",
      "callout-warning-title": "Warning",
      "callout-important-title": "Important",
      "callout-caution-title": "Caution",
      "code-summary": "Code",
      "code-tools-menu-caption": "Code",
      "code-tools-show-all-code": "Show All Code",
      "code-tools-hide-all-code": "Hide All Code",
      "code-tools-view-source": "View Source",
      "code-tools-source-code": "Source Code",
      "tools-share": "Share",
      "tools-download": "Download",
      "code-line": "Line",
      "code-lines": "Lines",
      "copy-button-tooltip": "Copy to Clipboard",
      "copy-button-tooltip-success": "Copied!",
      "repo-action-links-edit": "Edit this page",
      "repo-action-links-source": "View source",
      "repo-action-links-issue": "Report an issue",
      "back-to-top": "Back to top",
      "search-no-results-text": "No results",
      "search-matching-documents-text": "matching documents",
      "search-copy-link-title": "Copy link to search",
      "search-hide-matches-text": "Hide additional matches",
      "search-more-match-text": "more match in this document",
      "search-more-matches-text": "more matches in this document",
      "search-clear-button-title": "Clear",
      "search-text-placeholder": "",
      "search-detached-cancel-button-title": "Cancel",
      "search-submit-button-title": "Submit",
      "search-label": "Search",
      "toggle-section": "Toggle section",
      "toggle-sidebar": "Toggle sidebar navigation",
      "toggle-dark-mode": "Toggle dark mode",
      "toggle-reader-mode": "Toggle reader mode",
      "toggle-navigation": "Toggle navigation",
      "crossref-fig-title": "Figure",
      "crossref-tbl-title": "Table",
      "crossref-lst-title": "Listing",
      "crossref-thm-title": "Theorem",
      "crossref-lem-title": "Lemma",
      "crossref-cor-title": "Corollary",
      "crossref-prp-title": "Proposition",
      "crossref-cnj-title": "Conjecture",
      "crossref-def-title": "Definition",
      "crossref-exm-title": "Example",
      "crossref-exr-title": "Exercise",
      "crossref-ch-prefix": "Chapter",
      "crossref-apx-prefix": "Appendix",
      "crossref-sec-prefix": "Section",
      "crossref-eq-prefix": "Equation",
      "crossref-lof-title": "List of Figures",
      "crossref-lot-title": "List of Tables",
      "crossref-lol-title": "List of Listings",
      "environment-proof-title": "Proof",
      "environment-remark-title": "Remark",
      "environment-solution-title": "Solution",
      "listing-page-order-by": "Order By",
      "listing-page-order-by-default": "Default",
      "listing-page-order-by-date-asc": "Oldest",
      "listing-page-order-by-date-desc": "Newest",
      "listing-page-order-by-number-desc": "High to Low",
      "listing-page-order-by-number-asc": "Low to High",
      "listing-page-field-date": "Date",
      "listing-page-field-title": "Title",
      "listing-page-field-description": "Description",
      "listing-page-field-author": "Author",
      "listing-page-field-filename": "File Name",
      "listing-page-field-filemodified": "Modified",
      "listing-page-field-subtitle": "Subtitle",
      "listing-page-field-readingtime": "Reading Time",
      "listing-page-field-wordcount": "Word Count",
      "listing-page-field-categories": "Categories",
      "listing-page-minutes-compact": "{0} min",
      "listing-page-category-all": "All",
      "listing-page-no-matches": "No matching items",
      "listing-page-words": "{0} words"
    },
    metadata: {
      lang: "en",
      "fig-responsive": true,
      "quarto-version": "99.9.9",
      julia: { some_key: "some value", exeflags: [ "--project=." ] },
      project: {},
      format: { html: {}, pdf: { "include-in-header": [Array] }, typst: {} },
      title: "Quarto test"
    },
    resolveFormat: [Function: resolveFormat],
    formatExtras: [AsyncFunction: formatExtras],
    extensions: { book: { multiFile: true } },
    mergeAdditionalFormats: [Function (anonymous)]
  },
  projectDir: "/Users/krumbiegel/temp/quarto-tests/projecttest",
  cwd: "/Users/krumbiegel/temp/quarto-tests/projecttest",
  params: undefined,
  quiet: undefined,
  previewServer: undefined,
  handledLanguages: [ "mermaid", "dot" ],
  projectType: undefined
}
```